packages/web/src/pages/search-page/components/desktop/SearchPageContent.js

```diff
@@ -415,7 +421,11 @@ class SearchPageContent extends Component {
                 }
                 loadMore={(offset, limit) =>
                   this.props.dispatch(
-                    tracksActions.fetchLineupMetadatas(offset, limit)
+                    tracksActions.fetchLineupMetadatas(offset, limit, false, {
```

Comment:
interesting! was this stuff all broken even on web? or maybe looks like it was computed in sagas but not native compat?

Reply:
yeah the latter! previously it was computed using the window.location